src/ansys/geometry/core/tools/measurement_tools.py

```diff
@@ -58,7 +53,7 @@ def distance(self) -> Distance:
         return self._distance
 
     @classmethod
-    def _from_distance_response(cls, response: MinDistanceBetweenObjectsResponse) -> "Gap":
+    def _from_distance_response(cls, distance: Real) -> "Gap":
```

Comment:
At this point, I would probably remove all this method -- there is no need anymore really the way I see it,